Shared/src/shared/models/card/Card.java

```diff
@@ -146,6 +146,13 @@ public boolean isSameAs(String cardName) {
         return name.equalsIgnoreCase(cardName);
     }
 
+    public boolean IDSameAs(String cardID){
```

Comment:
Maybe I'm confused and thinking about a commit that did not get merged. 

BUT, don't we have a function somewhere that does this?

Reply:
a similar function was created in the old fix-replace-bug PR, that that PR was never merged in.